pkg/asset/ignition/bootstrap/common.go

```diff
@@ -612,6 +618,7 @@ func (a *Common) addParentFiles(dependencies asset.Parents) {
 		&tls.MCSCertKey{},
 		&tls.ServiceAccountKeyPair{},
 		&tls.JournalCertKey{},
+		&tls.IronicVirtualMediaTLSCert{},
```

Comment:
Thought: let us strategically call this just IronicTLSCert (and pkg/asset/tls/ironictls.go). First, because it requires less context to understand  ("VirtualMedia" is not a common concept). Second, because I expect us to eventually reuse the certificate for Ironic itself.